### DISCUSSION TITLE
addresses should have prefix iota.

### DISCUSSION THREAD

**Discussion by Floody500** [iotaledger/firefly#8548] [Help & Support]
I cant sent Iota tokens to another wallet address as this prefix error message comes up. what does this mean?

**Comment by begonaalvarezd** [comment]
What are you trying to do? Send iota tokens from firefly to where? 
If you for example want to send to an exchange, you have to create your IOTA wallet in the exchange and use the exchange provided address. If you want to send your funds to EVM, you must change the selected network in the send popup and change to IOTA EVM so that you an enter your eth address

**Comment by Floody500** [comment]
Hi. I tried to send to my coinbase wallet and also meta mask. On both occasions the error came up - addresses should have prefix itota.- so the issue is with the receiver. I will investigate receiving iota on the coinbase exchange.

Thanks for your support


Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by begonaalvarezd** [reply]
Metamask is EVM and uses ethereum addresses, not iota ones. If you want to send funds to the IOTA EVM you need to change the selected network to IOTA EVM 
![image](https://github.com/iotaledger/firefly/assets/3624944/c9f44948-2810-46f3-a4d9-749d3c177df0)
For exchanges you need to follow their instructions to deposit funds into their wallets

**Comment by Floody500** [comment]
Hi. Thanks for ongoing support. I will follow instruction.

Kind Regards

Mike

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
